common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerActivity.java

```diff
@@ -26,14 +26,18 @@
 import android.content.Intent;
 import android.os.Bundle;
 
-import androidx.localbroadcastmanager.content.LocalBroadcastManager;
-
-import com.microsoft.identity.common.adal.internal.AuthenticationConstants;
+import com.microsoft.identity.common.DataBagUtil;
+import com.microsoft.identity.common.java.util.ported.DataBag;
```

Comment:
What does the "ported" namespace convey?

Reply:
Classes that exists in Android or newer version of java.